docs/src/css/custom.css

```
@@ -1044,9 +970,132 @@ aside.theme-doc-sidebar-container {
+.apis-page .request-body-header {
+  align-items: center;
+  margin-bottom: 12px !important;
+}
+
+/* "required" badge next to "Body" heading — de-emphasise to secondary info */
+.apis-page .request-body-required {
+  font-size: 0.65rem !important;
+  font-weight: 600 !important;
+  text-transform: uppercase;
+  letter-spacing: 0.05em;
+  border: 1px solid currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)